src/main/java/org/truffleruby/core/regexp/RegexpNodes.java

```diff
@@ -693,13 +693,28 @@ public DynamicObject source(DynamicObject regexp) {
     @CoreMethod(names = "to_s")
     public abstract static class ToSNode extends CoreMethodArrayArgumentsNode {
 
-        @TruffleBoundary
-        @Specialization
+        @Specialization(guards = "hasString(regexp)")
+        public DynamicObject toSFast(DynamicObject regexp) {
+            return createString(Layouts.REGEXP.getString(regexp));
+        }
+
+        @Specialization(guards = "!hasString(regexp)")
         public DynamicObject toS(DynamicObject regexp) {
+            Rope byteList = createRope(regexp);
```

Comment:
This should be named `rope`

Reply:
In keeping with the existing code style, it should be `final` as well.